modules/distribution/src/repository/resources/dbscripts/apimgt/oracle.sql

```diff
@@ -349,13 +351,13 @@ CREATE TABLE SP_FEDERATED_IDP (
 ALTER TABLE SP_FEDERATED_IDP ADD CONSTRAINT STEP_ID_CONSTRAINT FOREIGN KEY (ID) REFERENCES SP_AUTH_STEP (ID) ON DELETE CASCADE
 /
 CREATE TABLE SP_CLAIM_MAPPING (
-            ID INTEGER,
-            TENANT_ID INTEGER NOT NULL,
-            IDP_CLAIM VARCHAR (512) NOT NULL ,
+	    	ID INTEGER,
+	    	TENANT_ID INTEGER NOT NULL,
+	    	IDP_CLAIM VARCHAR (512) NOT NULL ,
             SP_CLAIM VARCHAR (512) NOT NULL ,
-            APP_ID INTEGER NOT NULL,
-            IS_REQUESTED VARCHAR(128) DEFAULT '0',
-            DEFAULT_VALUE VARCHAR(255),
+	   		APP_ID INTEGER NOT NULL,
+	    	IS_REQUESTED VARCHAR(128) DEFAULT '0',
+			DEFAULT_VALUE VARCHAR(255),
```

Comment:
Directly copied content from https://github.com/wso2/carbon-apimgt/tree/v5.0.4/features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql as apim version upgraded to v5.04